Makethird

```diff
@@ -3,9 +3,9 @@
 ifeq ($(USE_SYSTEM_LIBS),yes)
   USE_SYSTEM_FREETYPE := yes
   USE_SYSTEM_HARFBUZZ := yes
-  USE_SYSTEM_JBIG2DEC := yes
+  USE_SYSTEM_JBIG2DEC := no
   USE_SYSTEM_JPEGXR := no # not available
-  USE_SYSTEM_LCMS2 := no # lcms2mt is strongly preferred
+  USE_SYSTEM_LCMS2 := yes # lcms2mt is strongly preferred
```

Comment:
Is this enough to force the usage of system lcms2 lib? I tried this patch with version 1.16.1 and set USE_SYSTEM_LIBS to 1 during build. However, it still uses the lcms source code in thirdparty/lcms2.

Reply:
I think I figured out what the problem is. Line 8 has a trailing space after "yes" so it actually sets "USE_SYSTEM_LCMS2" to "yes " (notice the space here) rather than "yes". This makes the comparison `ifeq ($(USE_SYSTEM_LCMS2),yes)` fail.

Reply:
An oversight on my part, sorry.

Reply:
Can you actually make the change in your PR to fix this? I am not a member of the mupdf team so I don't have write access to your PR.